### PR TITLE
PCM-4460 code changes

### PR DESCRIPTION
**Pull request by gandhikeyur**
@vrathee @engineersamuel Please review.

Internal secure support associates need to be able to changes case owner and update sbr groups via PCM. These changes are related to that.

### CODE REVIEW

File: app/cases/views/edit.jade [hunk 1]
```diff
@@ -20,8 +20,8 @@ div.edit(ng-if='showCasePage()')
         span(id="rha-case-filedOn" translate)  Filed on {{$parent.CaseService.kase.created_date}} by {{$parent.CaseService.kase.created_by}}
       .row.row-very-short
         span {{'Assigned to'|translate}} {{$parent.CaseService.kase.owner}}
-        //a#cancelCaseDetailChanges.margin-left(ng-click="changeOwner()",translate='',
-        //ng-if='$parent.securityService.loginStatus.authedUser.is_internal && $parent.securityService.loginStatus.authedUser.is_secure_support_tech') {{'Edit'|translate}}
+        a#cancelCaseDetailChanges.margin-left(ng-click="changeOwner()",translate='',
+        ng-if='$parent.securityService.loginStatus.authedUser.is_internal && $parent.securityService.loginStatus.authedUser.is_secure_support_tech') {{'Edit'|translate}}
```
Comment (vrathee) [inline app/cases/views/edit.jade]: Since this is not related to SBR changes.. I guess this is intentional..

**Comment by vrathee** [comment]
Reviewed! Gave a comment...

File: app/cases/views/edit.jade (hunk 1 above)
Comment (gandhikeyur) [inline app/cases/views/edit.jade]: Yes, that's intentionally uncommented as we need to provide caseOwnerUpdate as well
